### ISSUE TITLE
Style/TrailingCommaInArguments with EnforcedStyleForMultiline: consistent_comma false positive

### ISSUE THREAD

**Issue by okuramasafumi** [rubocop-hq/rubocop#6614]
Style/TrailingCommaInArguments Cop works differently from documentation with `EnforcedStyleForMultiline: consistent_comma` configuration.

--------

## Expected behavior

Given file named `sample.rb`:

```ruby
# frozen_string_literal: true

'foobar'.downcase
        .sub('bar', 'buzz')
```

and `.rubocop.yml` file below:

```yaml
Style/TrailingCommaInArguments:
  EnforcedStyleForMultiline: consistent_comma
```

It's expected that when I run `rubocop` command, there is no warnings.

## Actual behavior

Output from `rubocop` command is below:

```
Inspecting 1 file
C

Offenses:

sample.rb:4:21: C: Style/TrailingCommaInArguments: Put a comma after the last parameter of a multiline method call.
        .sub('bar', 'buzz')
                    ^^^^^^
```

## Steps to reproduce the problem

I described reproduction steps above.

## RuboCop version

```
rubocop -V
0.61.1 (using Parser 2.5.3.0, running on ruby 2.6.0 x86_64-darwin15)
```

## Ruby version
```
ruby -v
ruby 2.6.0p0 (2018-12-25 revision 66547) [x86_64-darwin15]
```

**Comment by okuramasafumi** [comment]
Soon after I created this issue, I noticed it's a duplication of https://github.com/rubocop-hq/rubocop/issues/6571 and already fixed. Sorry!
